ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh

```diff
@@ -337,7 +337,7 @@ if [[ "$HOSTED_CP" == "true" ]]; then
     # ensure the SC is not for ibm usage so that it could support the latest version of the hosted cluster
     for ps in $psList ; do
       topology=$(ocm get /api/clusters_mgmt/v1/provision_shards/${ps} | jq -r '.hypershift_config.topology')
-      if [[ "$topology" == "dedicated"  ]] ; then
+      if [[ "$topology" == "dedicated"  ]] || [[ "$topology" == "dedicated-v2" ]] ; then
```

Comment:
i think `$topology == *dedicated*` maybe better if we have more dedicated topology

Reply:
done